### DISCUSSION TITLE
Function for setting the names of individual cycles/timers and their parameters

### DISCUSSION THREAD

**Discussion by ghost** [Hanziness/AnotherPomodoro#190] [Ideas & feature requests]
[Cycle name [timer name [duration, theme color, ...

Example cycle:
[Monday [work [60m] short rest [7m] research [2h] short rest [15m] learning [30m] long rest [2h]], ...]

**Comment by Hanziness** [comment]
I wanted to do a programmable schedule, but something like this is simply out of scope for me now as it requires a lot of parsing logic and restructuring of the scheduling logic :/

Maybe a drag-and-drop reordering of the work, short pause and long pause sections could be done.
